examples/terra-contract/Developing.md

```diff
@@ -52,18 +59,20 @@ produce an extremely small build output in a consistent manner. The suggested wa
 to run it is this:
 
 ```sh
+cd path/to/cargo/root
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.4
+  cosmwasm/rust-optimizer:0.12.6
 ```
 
 Or, If you're on an arm64 machine, you should use a docker image built with arm64.
 ```sh
+cd path/to/cargo/root
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer-arm64:0.12.4
+  cosmwasm/rust-optimizer-arm64:0.12.6
 ```
```

Comment:
```
error: the lock file /code/Cargo.lock needs to be updated but --locked was passed to prevent this
If you want to try to generate the lock file without accessing the network, remove the --locked flag and use --offline instead.
```

I am getting this error. I think it's related to there not being a `Cargo.lock` inside `examples/terra-contract`.
The `Cargo.lock` is at the root of the repo, but doesn't get mounted into docker.